bokehjs/test/integration/annotations/legend.ts

```diff
@@ -48,4 +48,28 @@ describe("Legend annotation", () => {
 
     await display(p)
   })
+
+  it(`should display title correctly`, async () => {
+    const random = new Random(1)
+
+    const p = fig([600, 600])
```

Comment:
Does this figure have to be so large? The larger a figure or layout gets, the larger generated images are and the slower it's to test. The size has to be optimized per test (or class of tests) basis, i.e. as small as possible, but large enough so that we can discern relevant visual features with a naked eye.